airbyte-webapp/src/packages/cloud/views/credits/CreditsPage/components/UsagePerConnectionTable.tsx

```diff
@@ -105,6 +114,152 @@ const UsagePerConnectionTable: React.FC<UsagePerConnectionTableProps> = ({ credi
+                isActive={sortBy === "sourceConnectionName"}
+                isAscending={sortOrder === SortOrderEnum.ASC}
+              >
+                <FormattedMessage id="credits.source" />
+              </SortableTableHeader>
+            ),
+            meta: {
+              thClassName: classNames(styles.header, styles.light),
+              responsive: true,
+            },
+            cell: (props) => (
+              <NavLink
+                to={`/${RoutePaths.Workspaces}/${workspaceId}/${RoutePaths.Source}/${props.row.original.sourceId}`}
+              >
+                <FlexContainer direction="row" alignItems="center">
+                  <ConnectorIcon icon={props.row.original.sourceIcon} />
+                  <Text size="sm" className={styles.cellText}>
+                    {props.cell.getValue()}
+                  </Text>{" "}
+                </FlexContainer>
+              </NavLink>
+            ),
+          }),
+          columnHelper.display({
+            id: "arrow",
+            cell: () => <ArrowRightIcon />,
+            meta: {
+              thClassName: classNames(styles.header, styles.light),
+              responsive: true,
+            },
+          }),
+          columnHelper.accessor("destinationConnectionName", {
+            header: () => (
+              <SortableTableHeader
+                onClick={() => onSortClick("destinationConnectionName")}
+                isActive={sortBy === "destinationConnectionName"}
+                isAscending={sortOrder === SortOrderEnum.ASC}
+              >
+                <FormattedMessage id="credits.destination" />
+              </SortableTableHeader>
+            ),
+            meta: {
+              thClassName: classNames(styles.header, styles.light),
+              responsive: true,
+            },
+            cell: (props) => (
+              <NavLink
+                to={`/${RoutePaths.Workspaces}/${workspaceId}/${RoutePaths.Destination}/${props.row.original.destinationId}`}
+              >
+                <FlexContainer direction="row" alignItems="center">
+                  <ConnectorIcon icon={props.row.original.destinationIcon} />
+                  <Text size="sm">{props.cell.getValue()}</Text>
+                </FlexContainer>{" "}
+              </NavLink>
+            ),
+          }),
+          columnHelper.display({
+            id: "schedule",
+            header: () => (
+              <SortableTableHeader
+                onClick={() => {
+                  return null;
+                }}
+                isActive={false}
+                isAscending={false}
+              >
+                <FormattedMessage id="credits.schedule" />
+              </SortableTableHeader>
+            ),
+            cell: () => (
+              <FlexContainer className={styles.cell} alignItems="center">
+                <Text size="sm" className={styles.cellText}>
+                  Every 6 hours
```

Comment:
```suggestion
                  formatMessage({id: "credits.unknownColumnValue" })
```

Where `credits.unknownColumnValue: "Not available"` or something similar. Just to avoid confusion if anyone is testing this, didn't read the PR description, and wonders why their connection schedules are all the same 🙂

Reply:
Sure thing.

I swapped it out for "PLACEHOLDER!!" (to make it even more obvious it's a placeholder, since there wouldn't be an "unknown value" here in the final product)